src/github.com/getlantern/flashlight/app/settings.go

```diff
@@ -145,6 +145,14 @@ func (s *Settings) checkBool(data map[string]interface{}, name string, f func(bo
 	}
 }
 
+func (s *Settings) checkInt(data map[string]interface{}, name string, f func(int)) {
+	if v, ok := data[name].(int); ok {
```

Comment:
I switched this to cast it to an int @xiam, as using float64 made the tests fail. What was the thought behind float64 versus int?